recipes/r-spectrum/meta.yaml

```
@@ -0,0 +1,82 @@
+{% set version = '1.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-spectrum
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/Spectrum_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/Spectrum/Spectrum_{{ version }}.tar.gz
+  sha256: dcb11ed6051f6e147279cf27a7716a7b3d00f50fe764b8b7996c7e05cc686858
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip                # [win]
+    - msys2-conda-epoch ==20160418  # [win]
+    - cross-r-base {{ r_base }}     # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-clusterr
+    - r-rfast
+    - r-diptest
+    - r-ggplot2
+  run:
+    - r-base
+    - r-clusterr
+    - r-rfast
+    - r-diptest
+    - r-ggplot2
+
+test:
+  commands:
+    - $R -e "library('Spectrum')"           # [not win]
+    - "\"%R%\" -e \"library('Spectrum')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=Spectrum
+  license: AGPL-3.0-only
+  summary: A self-tuning spectral clustering method for single or multi-view data. 'Spectrum'
+    uses a new type of adaptive density aware kernel that strengthens connections in
+    the graph based on common nearest neighbours. It uses a tensor product graph data
+    integration and diffusion procedure to integrate different data sources and reduce
+    noise. 'Spectrum' uses either the eigengap or multimodality gap heuristics to determine
+    the number of clusters. The method is sufficiently flexible so that a wide range
+    of Gaussian and non-Gaussian structures can be clustered with automatic selection
+    of K.
+  license_family: AGPL
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/AGPL-3'
+
+extra:
+  recipe-maintainers:
+    - nigiord
```

Please include R team:

```suggestion
    - conda-forge/r
    - nigiord
```